content/influxdb/v2.0/tools/repl.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Interactive Flux REPL
+seotitle: Interactive Flux REPL
+description: Use the Flux REPL to test queries and interact with data.
+influxdb/v2.0/tags: [flux]
+menu:
+  influxdb_2_0:
+    parent: Tools
+weight: 104
+---
+
+Use the Flux REPL (Read–Eval–Print Loop) to execute Flux scripts and interact with InfluxDB and other data sources.
+[Build the REPL](#build-the-repl) from the Flux source code.
+
+## Build the REPL
+
+See the [Flux repository README](https://github.com/influxdata/flux#requirements) for instructions on building the REPL.
+
+## Use the REPL
+
+Once built, open a new REPL session with:
+
+```sh
+./flux repl
+```
+
+To query data from InfluxDB, provide the organization, token, and host parameters to the [`from()` function](/v2.0/reference/flux/stdlib/built-in/inputs/from/).
```

Comment:
Include an example:

```js
from(bucket: "example-bucket", org: "example-org", token: "My5uP3rS3cRetT0k3n")
```

You may also want to note that they can query remote InfluxDB hosts as well by including the host parameter.

```js
from(bucket: "example-bucket", host: "http://localhost:9999" org: "example-org", token: "My5uP3rS3cRetT0k3n")
```

_Using `localhost:9999` will let them use the URL picker to update the URL._